wasm-node/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "wasm-node"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+celestia-node = { workspace = true }
+celestia-types = { workspace = true }
+
+anyhow = "1.0.71"
+console_error_panic_hook = "0.1.7"
+js-sys = "0.3.64"
+libp2p = "0.52.3"
```

Comment:
should we maybe specify libp2p version on workspace level?

Reply:
It's getting pretty ubiquitous so I think it's a good idea.
I've made the change in b3176aad0df9d866650af6c224893cc97051739b, can split it out if this PR is getting too big.